removed

```diff
@@ -495,5 +495,10 @@
     "reason": "Repository does not exist",
     "removal_type": "remove",
     "link": "https://github.com/hacs/integration/issues/2080"
+  },
+  {
+    "repository": "jomwells/ambihue",
+    "reason": "It's abandoned",
+    "removal_type": "replaced"
```

Comment:
Missing link to information about why

Reply:
Can it be link to this PR or to replaced repo?

Reply:
yes